packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -806,105 +785,8 @@ test("Case-sensitive tag has capitalization preserved", function() {
   compilesTo('<svg><linearGradient id="gradient"></linearGradient></svg>');
 });
 
-// QUnit.skip("svg can live with hydration", function() {
```

Comment:
Moved to `updating-test.ts` as INUR, renamed `<svg> tag and expression as sibling`